### PR TITLE
Vue Router

### PR DESCRIPTION
**Pull request by Marcosnr97**
### Summary
- Added routes for login and register 

### Trello Card
https://trello.com/c/arSquYyn/18-vue-router

### CODE REVIEW

File: wolox-books/src/router.js [hunk 1]
```diff
@@ -6,17 +6,18 @@ Vue.use(Router)
 export default new Router({
   routes: [
     {
-      path: '/',
+      path: '/register',
       name: 'register',
```
Comment (BJordi) [inline wolox-books/src/router.js]: Add a const object called `routes` or something like that to have all the routes names. Then, you can export it and use it somewhere else, and if you have to change a name you'll do it in just one place.

File: wolox-books/src/router.js [hunk 2]
```diff
@@ -6,17 +6,18 @@ Vue.use(Router)
...
-      component: () => import(/* webpackChunkName: "register" */ './views/Register.vue')
+      component: () => import('./views/Register.vue')
```
Comment (BJordi) [inline wolox-books/src/router.js]: Add the chunk name:
`component: () => import(/* webpackChunkName: "register" */ './views/Register.vue')`

File: wolox-books/src/router.js [hunk 3]
```diff
@@ -6,17 +6,18 @@ Vue.use(Router)
...
+    },
+    {
+      path: '/login',
+      name: 'login',
+      component: () => import('./views/Login.vue')
```
Comment (BJordi) [inline wolox-books/src/router.js]: Do the same here, use that `routes` for the name and add the chunk name.

File: wolox-books/src/router.js [hunk 4]
```diff
@@ -6,17 +6,18 @@ Vue.use(Router)
...
     },
     {
-      path: '/about',
-      name: 'about',
-      // route level code-splitting
-      // this generates a separate chunk (about.[hash].js) for this route
-      // which is lazy-loaded when the route is visited.
-      component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
+      path: '/',
+      redirect: { name: 'login' }
```
Comment (BJordi) [inline wolox-books/src/router.js]: Use `routes` here too.

File: wolox-books/src/views/Register.vue [hunk 5]
```diff
@@ -34,7 +34,7 @@
       .sign-up-container
         button.main-button.text-xsmall(@click='Submit')
           | Sign up
-      button.secondary-button.text-xsmall.white
+      router-link.secondary-button.text-xsmall.white(to='`/login`')
```
Comment (BJordi) [inline wolox-books/src/views/Register.vue]: Remove the grave accent, just type `to='/login'`

**Review by BJordi — CHANGES_REQUESTED** [review]

File: wolox-books/src/views/Register.vue [hunk 6]
```diff
@@ -32,14 +32,16 @@
           span.error(v-show='missingPassword').text-xxxsmall
             | Password is required
       .sign-up-container
-        button.main-button.text-xsmall(@click='signIn')
+        button.main-button.text-xsmall(@click='Submit')
```
Comment (BJordi) [inline wolox-books/src/views/Register.vue]: `Submit` -> `submit`

File: wolox-books/src/views/Register.vue [hunk 7]
```diff
@@ -32,14 +32,16 @@
...
           | Sign up
-      button.secondary-button.text-xsmall.white
+      router-link.secondary-button.text-xsmall.white(to='/login')
```
Comment (BJordi) [inline wolox-books/src/views/Register.vue]: `:to='router.login'`

**Review by BJordi — CHANGES_REQUESTED** [review]

File: wolox-books/src/views/Register.vue [hunk 8]
```diff
@@ -31,7 +31,7 @@
   .sign-up-container
     button.main-button.text-xsmall(@click='submit' type='button')
       | Sign up
-  button.secondary-button.text-xsmall.white(type='button')
+   router-link.secondary-button.text-xsmall.white(:to='router.login')
```
Comment (LauchaBo) [inline wolox-books/src/views/Register.vue]: Change the destination to '/login'

File: wolox-books/src/views/Register.vue [hunk 9]
```diff
@@ -146,7 +146,10 @@ export default {
 }
 
 .secondary-button {
+  align-items: center;
+  display: flex;
```
Comment (LauchaBo) [inline wolox-books/src/views/Register.vue]: Create a class 'center' outside this view in 'src/scss/commons/display.scss' and use that class in the pug code. Don't forget to add the new scss file to 'application.scss'

**Review by LauchaBo — CHANGES_REQUESTED** [review]

File: wolox-books/src/views/Register.vue (hunk 8 above)
Comment (Marcosnr97) [reply wolox-books/src/views/Register.vue]: Check this comment from @BJordi https://github.com/wolox-training/mr-net-core/pull/56#discussion_r320092558 . :thinking:

Comment (LauchaBo) [reply wolox-books/src/views/Register.vue]: I talked to @BJordi and it should be written this way, as it is the one recommended in the vue docs:

`router-link(:to='{ name: routes.login}')`

Adapt 'name' in router.js to routes.login so it works with the new format

**Review by BJordi — APPROVED** [review]

**Pull request merged**